packages/core/src/utils/ai/utils.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * Shared utils for AI integrations (OpenAI, Anthropic, Verce.AI, etc.)
+ */
+import type { Span } from '../../types-hoist/span';
+import {
+  GEN_AI_USAGE_INPUT_TOKENS_ATTRIBUTE,
+  GEN_AI_USAGE_OUTPUT_TOKENS_ATTRIBUTE,
+  GEN_AI_USAGE_TOTAL_TOKENS_ATTRIBUTE,
+} from './gen-ai-attributes';
+/**
+ * Maps AI method paths to Sentry operation name
+ */
+export function getFinalOperationName(methodPath: string): string {
```

Comment:
I don't know if the develop docs are outdated on this one but this method seems to use other attribute names as defined here: https://develop.sentry.dev/sdk/telemetry/traces/modules/ai-agents/#common-span-attributes

Also checked here: https://getsentry.github.io/sentry-conventions/generated/attributes/gen_ai.html